charts/prometheus-fastly-exporter/values.yaml

```diff
@@ -21,7 +21,8 @@ image:
   tag: v7.2.4
   pullPolicy: IfNotPresent
 
-token: ""
+# API token Fastly - https://docs.fastly.com/en/guides/using-api-tokens#creating-api-tokens
+token: "token"
```

Comment:
It shoudl also be possible to use the token from an existing secret.

Reply:
Added new `envTokenFromSecret` variable